projects/box-office/get_movie_data.py

```diff
@@ -13,38 +13,41 @@
 ]
 
 
-# [
-#     {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36"},
-#     {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.47 Safari/537.36'},
-#     {"User-Agent": 'Mozilla/5.0 (Windows; U; Windows NT 5.1; it; rv:1.8.1.11) Gecko/20071127 Firefox/2.0.0.11'},
-#     {"User-Agent": 'Mozilla/5.0 (X11; U; Linux i686; en-US; rv:1.8.0.12) Gecko/20070731 Ubuntu/dapper-security Firefox/1.5.0.12'}
-# ]
-
 def movie_data(filmID):
+    """
+    Visits weekend info page on BoxOfficeMojo for a particular film and
+    scrapes some general and detailed data.
+    filmID: BoxOfficeMojo-specific film url id param in the format ?id=xxx.htm
+    return: two separate lists of data; one-line for general information,
+        and possibly multiple lines for detailed information.
+    """
+
     url = "http://www.boxofficemojo.com/movies/?page=weekend&" + filmID + "&adjust_yr=2015&p=.htm"
     headers = choice(HEADERS)
     page = Request(url, headers=headers)
 
     soup = BeautifulSoup(urlopen(page).read(), "lxml", from_encoding="utf-8")
     body = soup.find('div', {'id': 'body'})
-    
+
     if len(body.findAll(text=re.compile('Invalid Movie ID Specified.'))) > 0:
         return [], []
 
     general = [filmID]
     detailed = []
 
     # Domestic Total Gross
-    domestic_total = body.find(string=re.compile("Domestic Total"))
+    dom = re.compile("Domestic Total")
```

Comment:
Be careful with abbreviations. Usually in production code abbreviations are being used less frequently.